public/app/features/panel/panel_ctrl.ts

```diff
@@ -260,9 +260,9 @@ export class PanelCtrl {
     });
   }
 
-  addToPanelList() {
+  copyPanel() {
     store.set(LS_PANEL_COPY_KEY, JSON.stringify(this.panel.getSaveModel()));
-    appEvents.emit('alert-success', ['Panel temporarily added to panel list']);
+    appEvents.emit('alert-success', ['Panel copied. Open new panel to paste']);
```

Comment:
We need something better here I think. How about *Panel copied, open Add Panel to paste* or similar?

Reply:
I like that, sounds more clear and gives user a direct tip on where / how to paste.